ee/app/livechat-enterprise/server/lib/Helper.js

```diff
@@ -151,6 +154,21 @@ export const updatePredictedVisitorAbandonment = () => {
 	}
 };
 
+export const updateQueueInactivityTimeout = () => {
+	const queueAction = settings.get('Livechat_max_queue_wait_time_action');
```

Comment:
Yup, I thought about that but in this specific case we can't, since that function is called on the changes of 2 different, dependant, settings. So, instead of having to call the function like ths: `updateQueueInactivityTimeout(undefined, valueOfUpdatedSetting) or with (valueOfUpdatedSetting, undefined)` depending on which setting was updated, I preferred to fetch the value on each call. Anyways, this should only be called when the setting is updated